ontology/coap.ttl

```diff
@@ -0,0 +1,165 @@
+@prefix : <https://www.w3.org/2019/wot/coap#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <http://schema.org/> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@base <https://www.w3.org/2019/wot/coap> .
+
+<https://www.w3.org/2019/wot/coap> rdf:type owl:Ontology ;
+                                    dct:author [ rdf:type schema:Person ;
+                                                 schema:affiliation _:genid4 ;
+                                                 schema:name "Klaus Hartke"
+                                               ] ;
+                                    dct:contributor [ rdf:type schema:Person ;
+                                                      schema:affiliation [ rdf:type schema:Organization ;
+                                                      schema:name "Invited Expert" ;
+                                                      schema:url "https://github.com/JKRhb/"
+                                                    ] ;
+                                                 schema:name "Jan Romann"
+                                               ] ;
+                                    vann:preferredNamespacePrefix "cov" ;
+                                    vann:preferredNamespaceUri : ;
+                                    rdfs:comment "Vocabulary to represent CoAP messages in RDF."@en ;
+                                    rdfs:label "CoAP in RDF"@en .
```

Comment:
The [HTTP Vocabulary in RDF 1.0](https://w3c.github.io/wot-binding-templates/bindings/protocols/http/index.html) describes itself as follows (emphasize mine):

> This document provides a representation of the HTTP vocabulary in the Resource Description Framework (RDF), to allow quality assurance tools to **record the HTTP headers that have been exchanged between a client and a server**.

In other words, it provides a way to convert exchanged HTTP messages into RDF graphs. With a bit of imagination, this can also be (ab)used to serialize HTTP messages that haven't yet been exchanged, but the focus is clearly on serializing HTTP headers.

In the case of the vocabulary in the CoAP Binding document, the vocabulary is *not* intended for serializing exchanged CoAP options into RDF. The vocabulary instead is used to describe a CoAP server's capabilities and expected parameters for a WoT operation.

I therefore feel it is inappropriate to call this vocabulary "CoAP in RDF".

Reply:
Thank you for your feedback on this. I took over the old ontology title for now, for lack of a better alternative. Would something like "CoAP Binding Ontology" be more appropriate? Or do you have something in mind that would be a better fit here?

Reply:
Since the vocabulary is already fully documented in `bindings/protocols/coap/index.html`, why not just move `ontology/coap.ttl` to `bindings/protocols/coap/coap.ttl` and delete `ontology/coap.html`?

Reply:
The HTML document for this vocabulary is `bindings/protocols/coap/index.html`. And [the title is `Web of Things (WoT) CoAP Binding Template`](https://github.com/w3c/wot-binding-templates/blob/5a681062759b023b2ed3a1129a2df5b6dde8c0f8/bindings/protocols/coap/index.html#L51). (Although I'd agree that "CoAP Vocabulary for the Web of Things" would probably be a better title for both the vocabulary and `bindings/protocols/coap/index.html`.)